packages/app/components/item/searchItem.tsx

```diff
@@ -52,19 +52,17 @@ export const SearchItem: React.FC<Props> = ({ onSelect, placeholder }) => {
           style={{ position: 'relative', height: 'auto', flexDirection: 'row' }}
         >
           <RInput
-            onChangeText={(text) => {
+            onChangeText={(text: string) => {
               setSearchString(text);
-              // @ts-expect-error
               dispatch(fetchItemsSearchResults(text));
               setShowSearchResults(true);
             }}
             placeholder={placeholder ?? 'Type here to search'}
             value={searchString}
             fontSize={14}
-            width={'100%'}
+            width="100%"
             borderRadius={4}
-            paddingVertical={16}
-            paddingHorizontal={8}
+            padding="16px 8px"
```

Comment:
we are moving away from px sized styles as they work on web but not on native also FYI